README.md

```diff
@@ -437,6 +437,9 @@ Run `make test` to invoke the test suite, found in the test-suite subdirectory.
 - Iulia Tamas
 - Andrew Twyman
 
+## Contributors
```

Comment:
Hmm, the consensus in the Dropbox open-source team is the opposite.  "If copyright authors are listed in a LICENSE, COPYING, AUTHORS or similar file" evaluates to false, so no action is required.  The only place our contributors are listed is in git history, which will be attributed when I merge this.  Can you possibly put your compliance team in touch with us?  Do you know if it's the Authors list which is at issue here, or do they think the addition is required even if it's the first one.

Anyhow, I'd rather not let legal wrangling actually get in the way of a good contribution.  I'd be open to possibly adding a CONTRIBUTORS file, with a paragraph at the top explaining it's not intended to be complete (the complete list being the git history), but only present where required for legal purposes.  I'd also be open to clarifying/renaming the Authors section in the README to make it clear it's a way of acknowledging "top contributors" not a list of all contributors.